packages/brand-moment/src/BrandMoment.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode, ReactElement } from "react"
+import { Heading, Box, Paragraph } from "@kaizen/component-library"
+import { Button, ButtonProps } from "@kaizen/draft-button"
+import { SceneProps } from "@kaizen/draft-illustration"
+import { assetUrl } from "@kaizen/hosted-assets"
+import classnames from "classnames"
+import styles from "./BrandMoment.scss"
+
+type Props = {
+  backgroundColor: "blue" | "purple" | "green" | "red"
+  header: ReactNode
+  illustration: ReactElement<SceneProps>
+  heading: ReactNode
+  subheading?: ReactNode
+  body?: ReactNode
+  primaryAction: ButtonProps
+  secondaryAction?: ButtonProps
+  footer?: ReactNode
+}
+
+export const BrandMoment = (props: Props) => (
+  <div
+    className={classnames(styles.body, {
+      [styles.blue]: props.backgroundColor === "blue",
+      [styles.green]: props.backgroundColor === "green",
+      [styles.red]: props.backgroundColor === "red",
+    })}
+  >
+    <header className={styles.header}>{props.header}</header>
+    <main className={styles.main}>
+      <div className={styles.left}>
+        <div className={styles.leftInner}>{props.illustration}</div>
+      </div>
+      <div className={styles.right}>
+        <div className={styles.rightInner}>
+          {props.subheading && (
+            <Box mb={0.5}>
+              <Heading variant="heading-3" tag="h1">
+                {props.subheading}
+              </Heading>
+            </Box>
+          )}
+          <Box mb={1.5}>
+            <Heading variant="display-0" tag={props.subheading ? "h2" : "h1"}>
+              {props.heading}
+            </Heading>
+          </Box>
+          {props.body && (
+            <Box mb={1.5}>
+              <Paragraph variant="intro-lede">{props.body}</Paragraph>
```

Comment:
Need to check: if there are any implications (html validity, accessibility, cascade styling) of embedding these

Reply:
I initially liked the idea of wrapping in a paragraph _only_ when it's a literal string, but on second thoughts that'd break if you have text with some inline markup.

So we either always wrap or never wrap. Keen to hear @ActuallyACat's take

Reply:
Yeah.. I'm guessing you saw the work I ended up abandoning in the commit history. It also means if you pass in a `<Trans>` component from Lingui you will no longer get the Paragraph. I concluded it will just end up confusing consumers, and probably end up with some incorrect paragraph variants being used. 

There are other Kaizen components doing this already as well:
* GuidanceBlock
* HeroPanel
* GenericTile

Reply:
The big difference there is that each of those components have a prop with a name that suggests it should only contain text (`information.text`, `text.description` etc) whereas this explicitly has some non-text, and in the stories we've already got an example of wanting to pass in a block level element, which is semantically incorrect.

But on a practical level, it seems fine. I'm not willing to die on the hill :) I would personally change it but I'll approve the PR anyway, I'm happy for you to do this if you're confident the API niceness is worth the weirdness of the html semantics

Reply:
You've convinced me.

Reply:
I took another look at this with fresh eyes and have come out with a solution that I'm happy with:

* All of the props that will be wrapped in some sort of Typography component are now part of a `text` object prop. This pattern is consistent the components I listed above as you pointed out.
* Props that are intended that have no or very little wrapping remain - `header`, `body`.
* There's now two ways to pass in body content. If you would like it to be wrapped in Typography styling, you  into the `text` object, otherwise it just goes into the `body` prop.
* This also provides a good solution for the comment about the footer here as well. As it stands, there is a `text.footer` prop which will be styled, and we have room for adding a base `footer` prop in the future if we need a way to avoid the CA logo. 

Side note: I renamed `heading` and `subheading` to `title` and `subtitle` to avoid confusion with `header`, and I noticed `title` has been used elsewhere in Kaizen as well.

Reply:
I'm a fan. Nice work!